pkg/common/juicefs.go

```diff
@@ -29,7 +29,7 @@ const (
 	JuiceFSCEImageEnv = "JUICEFS_CE_IMAGE_ENV"
 	JuiceFSEEImageEnv = "JUICEFS_EE_IMAGE_ENV"
 
-	DefaultCEImage = "juicedata/juicefs-fuse:ce-v1.1.0-beta2"
+	DefaultCEImage = "juicedata/juicefs-fuse:ce-nightly"
```

Comment:
Please don't change the image tag to ce-nightly. Because it's not production ready.

Reply:
ok, the doc for parallel migrate usage should be written in the pr ?

Reply:
I think we can create another PR to support. Thanks.